test/services/commits_runner_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class CommitsRunnerTest < ActiveSupport::TestCase
+  setup do
+    @repo = create(:repo)
+    BenchmarkPool.stubs(:enqueue)
```

Comment:
Instead of stubbing, perhaps we can look into the using the helps provided by sidekiq? 

https://github.com/mperham/sidekiq/wiki/Testing